src/functions/multiTestUpdate.ts

```diff
@@ -0,0 +1,40 @@
+import { UpdateTechRecordService } from "../services/UpdateTechRecordService";
+import { validateUpdateStatus } from "../validators/validateUpdateStatus";
+
+export const multiTestUpdate = (test: any): Array<Promise<any>> => {
```

Comment:
what is the difference between the multiTestUpdate and singleTestUpdate function?

Reply:
Ya know what, I've changed it enough now it probably does work for both